### PR TITLE
2934 update uri for stamps and metadata gerald

### PR DESCRIPTION
**Pull request by nutrina**
Fixes: https://github.com/passportxyz/passport/issues/3058

### CODE REVIEW

File: infra/lib/embed/index.ts [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+    comparisonOperator: "GreaterThanOrEqualToThreshold",
+    datapointsToAlarm: 1,
+    dimensions: {
+      FunctionName: lambdaName,
+    },
+    evaluationPeriods: 24,
+    metricName: "Errors",
+    name: metricAlarmName,
+    namespace: "AWS/Lambda",
+    period: 60 * 60, // 1 hours
```
Comment (nutrina) [inline infra/lib/embed/index.ts:113]: Period should be 60 seconds

File: infra/lib/embed/index.ts [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+  const lambdaErrorsAlarm = new aws.cloudwatch.MetricAlarm(metricAlarmName, {
+    tags: { ...defaultTags, Name: metricAlarmName },
+    alarmActions: [config.snsAlertsTopicArn],
+    okActions: [config.snsAlertsTopicArn],
+    comparisonOperator: "GreaterThanOrEqualToThreshold",
+    datapointsToAlarm: 1,
+    dimensions: {
+      FunctionName: lambdaName,
+    },
+    evaluationPeriods: 24,
```
Comment (nutrina) [inline infra/lib/embed/index.ts:109]: evaluation periods: 8 / 10

File: infra/lib/embed/index.ts [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+    },
+    {
+      dependsOn: [withLb],
+    }
+  );
+
+  const conditions: any = [
+    {
+      pathPattern: {
+        values: ["/test"],
```
Comment (nutrina) [inline infra/lib/embed/index.ts:150]: Change path to `/embed`

File: infra/lib/embed/index.ts [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+      httpRequestMethod: {
+        values: ["POST"],
+      },
+    },
+  ];
+
+  const targetPassportRule = new ListenerRule(`${config.name}-rule-lambda`, {
+    tags: { ...defaultTags, Name: `${config.name}-rule-lambda` },
+    listenerArn: config.httpsListener.arn,
+    priority: 12345,
```
Comment (nutrina) [inline infra/lib/embed/index.ts:163]: Change priority

File: infra/ops/lambda.ts [hunk 5]
```diff
@@ -52,123 +47,138 @@ export function createLambdaFunction(
...
-            Resource: "*",
-          },
-        ],
-      }),
-    });
+  // if (vpcId) {
+  // add VPC required permissions
+  const vpcPolicy = new aws.iam.Policy(`${functionArgs.name}-vpc-policy`, {
+    name: `${functionArgs.name}-vpc-policy`,
+    policy: JSON.stringify({
```
Comment (nutrina) [inline infra/ops/lambda.ts:63]: Remove the if: all our lambdas need to run within the VPC

File: infra/ops/lambda.ts [hunk 6]
```diff
@@ -28,19 +23,19 @@ export function createLambdaFunction(
             Service: "lambda.amazonaws.com",
           },
           Effect: "Allow",
-          Sid: `${name}LambdaAssumeRole`,
+          // Sid: `${functionArgs.name(n => n.replace("-", ""))}LambdaAssumeRole`,
```
Comment (nutrina) [inline infra/ops/lambda.ts:26]: Delete this line

File: infra/lib/layer/index.ts [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+      tags: {
+        ...defaultTags,
+        Name: layerBucketObjectName,
+      },
+    },
+    { dependsOn: [layerFolder] }
+  );
+
+  const layerName = `${config.name}-python-deps`;
+  const lambdaLayer = new aws.lambda.LayerVersion(
```
Comment (larisa17) [inline infra/lib/layer/index.ts:54]: Check if a different version is created every time or this one is updated with the new dependencies

Comment (nutrina) [reply infra/lib/layer/index.ts:53]: This has now been set up to create a new version every time.

**Pull request merged**